framework/web/Request.php

```diff
@@ -226,9 +226,6 @@ class Request extends \yii\base\Request
         // Microsoft:
         'Front-End-Https',
         'X-Rewrite-Url',
-
-        // RFC 7239:
-        'Forwarded',
```

Comment:
Removed because who has used it has to make changes and remove the header otherwise it can be attackable. Those who intend to use it, however, have the option to add it.